pdc/apps/common/viewsets.py

```diff
@@ -115,7 +127,11 @@ class ChangeSetDestroyModelMixin(mixins.DestroyModelMixin):
     def perform_destroy(self, obj):
         model_name = get_model_name_from_obj_or_cls(obj)
         obj_id = obj.id
-        obj_content = json.dumps(obj.export())
+        json_item = obj.export()
+        for k in json_item.keys():
+            if isinstance(json_item[k], datetime.date):
+                json_item[k] = str(json_item[k])
```

Comment:
Seems not work in export(), this should be get data functions.
I create a  DateToJsonFormat for this. Not sure if it was good to new a class for these few lines before.
And I think put the logic here is okay, because people maybe face the same issue if not add the  conversion here.

Reply:
Normally the `export()` doesn't take the extra argument (seems to be required only for one API). E.g. from push-targets (much easier to read IMO):
```python
def export(self):
    return {
        "name": self.name,
        "description": self.description,
        "host": self.host,
        "service": self.service.name,
    }
```

Reply:
Hmmm, maybe you are right, put the covert codes here is not good, because almost all the export data don't need to do such check. 
Change this part, just put codes in export function. 
Maybe someone can find what they need to do when face such json error.